deleted-files-list.json

```diff
@@ -1010,6 +1011,7 @@
     "tools/extensions/org.civicrm.search.basic/*",
     "tools/extensions/org.civicrm.search.multivalue.zip",
     "tools/extensions/org.civicrm.search.multivalue/*",
+    "tools/extensions/phpstorm/.phpstorm.meta.php",
```

Comment:
Annoying false-positive. I think there must be some commit that accidentally added and then deleted it, but we don't actually care about this file so shouldn't show an alert if you have it. If you run the script with the phpstorm extension enabled, then it leaves it out of the list.

Reply:
ok 🙃